tests/ui/parser/bad-let-else-statement.stderr

```diff
@@ -228,5 +228,31 @@ LL |         x
 LL ~     }) else {
    |
 
-error: aborting due to 17 previous errors
+error: right curly brace `}` before `else` in a `let...else` statement not allowed
+  --> $DIR/bad-let-else-statement.rs:169:28
+   |
+LL |     let bad = asm! {" nop "} else { return; };
+   |                            ^
+   |
+help: wrap the expression in parentheses
+   |
+LL |     let bad = (asm! {" nop "}) else { return; };
+   |               +              +
+
+error: right curly brace `}` before `else` in a `let...else` statement not allowed
+  --> $DIR/bad-let-else-statement.rs:183:25
+   |
+LL |             let x = a! {} else { return; };
+   |                         ^
+...
+LL |     b!(1); b!(2);
+   |            ----- in this macro invocation
+   |
+   = note: this error originates in the macro `b` (in Nightly builds, run with -Z macro-backtrace for more info)
+help: wrap the expression in parentheses
+   |
+LL |             let x = (a! {}) else { return; };
```

Comment:
in theory, `a!()` would also work, and in the case of say `write!{}` the more idiomatic suggestion, but I suppose it feels less wrong to suggest `(a!{})`.

Reply:
👍